src/components/Footer/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { PureComponent } from 'react'
+import cn from 'classnames'
+
+import logo from '../../assets/logo.svg'
+import sslBadge from '../../assets/ssl-badge.svg'
+
+export default class Footer extends PureComponent {
+  infoBlock = React.createRef()
```

Comment:
This variable is not used. please remove it

Reply:
Removed with next commit...